src/main/scala/io/confluent/examples/streams/algebird/CMSStore.scala

```diff
@@ -146,6 +145,8 @@ class CMSStore[T: CMSHasher](override val name: String,
     */
   private var cms: TopCMS[T] = cmsMonoid.zero
 
+  private var lastUpdate: Long = 0L
```

Comment:
For my understanding: Why do we need this now?

Reply:
Ah, I see it's a timestamp.  I'd suggest to rename `lastUpdate` to sth more descriptive like `timestampOfLastStateStoreUpdate` (or better/shorter).